website/docs/pushdown.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 6
+hide_table_of_contents: true
+---
+
+# Query pushdown
+
+**Query pushdown** is the method of optimizing a query by reducing the amount of data read and processed.
+It saves memory space, network bandwidth, and reduces the query execution time by not prefetching
+unnecessary data to the database management system.
+
+Initially FerretDB retrieved all data related to queried collection, and applies filters on its own, making
+it possible to implement complex logic safely and quickly.
+To make this process more efficient, we minimize the amount of incoming data, by applying proper SQL filters.
```

Comment:
Minor comment (feel free to ignore).

Maybe we can say something about what "proper" is? E.g. how we make it "proper"? I understand that we want to mention things briefly and link a blog post, but still, I think we should bring more clarity, just keep it summarized. As a reader of this text, I wonder where the filters are applied and when?

In principle, the next sentences cover it, but it looks too "secret" to me here.

Reply:
How can we improve this description? Should I add some simple query example and how is it evaluated or it's too much for docs?

Reply:
How about replacing `by applying proper SQL filters` to `by applying WHERE clause on SQL queries`? Is this more clear @rumyantseva?